lib/paper_trail-association_tracking.rb

```diff
@@ -10,9 +10,8 @@
 require "paper_trail_association_tracking/version_concern"
 
 if defined?(Rails)
-  require "paper_trail/frameworks/active_record"
   require "paper_trail_association_tracking/frameworks/rails"
-elsif defined?(ActiveRecord)
+else
```

Comment:
This change to else no longer makes sense and should be removed from the PR

Reply:
Yes go back to `elsif defined?(ActiveRecord)`. After you refactored the original changes with `config.to_prepare` then the suggestion no longer was applicable.